MetasysServices.Tests/MetasysClientTests.cs

```diff
@@ -282,26 +285,10 @@ public void TestGetObjectIdentifierBadRequestThrowsException()
             PrintMessage($"TestGetObjectIdentifierBadRequestThrowsException: {e.Message}");
         }
 
-        [Test]
-        public void TestGetObjectIdentifierBadResponseThrowsException()
-        {
-            httpTest.RespondWith($"\"{mockid}1\"");
-
-            var e = Assert.Throws<MetasysGuidException>(() =>
-                client.GetObjectIdentifier("fully:qualified/reference4"));
-
-            httpTest.ShouldHaveCalled($"https://hostname/api/v2/objectIdentifiers")
-                .WithVerb(HttpMethod.Get)
-                .Times(1);
-            var expected = new MetasysGuidException($"{mockid}1", new NullReferenceException());
-            Assert.AreEqual(expected.Message, e.Message);
-            PrintMessage($"TestGetObjectIdentifierBadResponseThrowsException: {e.Message}");
-        }
-
         [Test]
         public void TestGetObjectIdentifierNullResponseThrowsException()
         {
-            httpTest.RespondWith("null");
+            httpTest.RespondWithJson(JToken.Parse("null"));
```

Comment:
To test for null we need a JSON null which looks like `null` vs. a string with the word null which looks like `"null"`. The previous version of this test was just encoding the word `"null"` as a string.